src/api.ts

```diff
@@ -0,0 +1,8 @@
+import {TODO} from "./types";
+const URL = 'http://localhost:8080';
+
+export const fetchTodos = async (): Promise<TODO[]> => {
```

Comment:
Tohle tedy ještě přepíšeš, aby jsi použil RTK query? Jinak bych tady přidal ještě nějaký error handling.

Reply:
Aha zrovna jsem si všiml, že už jsi to udělal, tohle ti tady zůstalo asi od předchozí verze.